dyno-queues-redis/src/main/java/com/netflix/dyno/queues/demo/DynoQueueDemo.java

```diff
@@ -0,0 +1,85 @@
+package com.netflix.dyno.queues.demo;
+
+import com.netflix.dyno.demo.redis.DynoJedisDemo;
+import com.netflix.dyno.jedis.DynoJedisClient;
+import com.netflix.dyno.queues.DynoQueue;
+import com.netflix.dyno.queues.Message;
+import com.netflix.dyno.queues.redis.v2.QueueBuilder;
+import com.netflix.dyno.queues.shard.DynoShardSupplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+public class DynoQueueDemo extends DynoJedisDemo {
+
+    private static final Logger logger = LoggerFactory.getLogger(DynoQueue.class);
+    public DynoQueueDemo(String clusterName, String localRack) {
+        super(clusterName, localRack);
+    }
+
+    public DynoQueueDemo(String primaryCluster, String shadowCluster, String localRack) {
+        super(primaryCluster, shadowCluster, localRack);
+    }
+
+    public static void main(String[] args) throws IOException {
+        final String clusterName = args[0];
+
+        final DynoQueueDemo demo = new DynoQueueDemo(clusterName, "us-east-1e");
+        int numCounters = (args.length == 2) ? Integer.valueOf(args[1]) : 1;
+        Properties props = new Properties();
+        props.load(DynoQueueDemo.class.getResourceAsStream("/demo.properties"));
+        for (String name : props.stringPropertyNames()) {
+            System.setProperty(name, props.getProperty(name));
+        }
+
+        try {
+            demo.initWithRemoteClusterFromEurekaUrl(args[0], 8102);
+
+            demo.runSimpleQueueDemo(demo.client);
+            Thread.sleep(10000);
+
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        } finally {
+            demo.stop();
+            logger.info("Done");
+        }
+    }
+
+    private void runQueueDemoWithTimeOut(DynoJedisClient dyno) throws IOException {
+
+    }
+
+    private void runSimpleQueueDemo(DynoJedisClient dyno) throws IOException {
```

Comment:
Maybe best to differentiate between V1 and V2 tests in the function names itself (or any other way you see fit):
Eg: runSimpleV1QueueDemo()

Then we can remove this convention later once we consolidate the versions.

Reply:
makes sense. this is using v2. so will name it runSimpleV2QueueDemo.